### PR TITLE
feat(facade): add getMaxColumns and getMaxRows API on FWorksheet

### PR DESCRIPTION
**Pull request by fmwww**

### CODE REVIEW

File: packages/facade/src/apis/sheets/f-worksheet.ts [hunk 1]
```diff
@@ -90,4 +90,20 @@ export class FWorksheet {
             colWidth: width,
         } as ISetWorksheetColWidthMutationParams);
     }
+
+    /**
+     * Returns the current number of columns in the sheet, regardless of content.
+     * @return The maximum width of the sheet
```
Comment (jikkai) [inline packages/facade/src/apis/sheets/f-worksheet.ts:96]: Maybe it's not related to the width or the height?

**Review by jikkai — CHANGES_REQUESTED** [review]

File: packages/facade/src/apis/sheets/f-worksheet.ts (hunk 1 above)
Comment (fmwww) [reply packages/facade/src/apis/sheets/f-worksheet.ts:96]: 我看GoogleSheet那边是这么定义的 [https://developers.google.com/apps-script/reference/spreadsheet/sheet#getmaxcolumns](url)

Comment (jikkai) [reply packages/facade/src/apis/sheets/f-worksheet.ts:96]: This might be an mistake, and insisting on using `width` and `height` could lead to confusion.

**Review by hexf00 — APPROVED** [review]

**Review by jikkai — APPROVED** [review]
LGTM

**Pull request merged**